<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>V4L Control GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="currentIndex" >
       <number>2</number>
      </property>
      <widget class="QWidget" name="info_tab" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground" >
        <bool>false</bool>
       </property>
       <attribute name="title" >
        <string>Device Info</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2" >
        <property name="fieldGrowthPolicy" >
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="1" >
         <layout class="QFormLayout" name="formLayout" >
          <property name="sizeConstraint" >
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="rowWrapPolicy" >
           <enum>QFormLayout::WrapLongRows</enum>
          </property>
          <property name="horizontalSpacing" >
           <number>5</number>
          </property>
          <property name="margin" >
           <number>5</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Device:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="device_label" >
            <property name="text" >
             <string/>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Driver:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="driver_label" >
            <property name="text" >
             <string/>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Card:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="card_label" >
            <property name="text" >
             <string/>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Bus Info:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLabel" name="bus_info_label" >
            <property name="text" >
             <string/>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Capabilities:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLabel" name="capabilities_label" >
            <property name="text" >
             <string/>
            </property>
            <property name="scaledContents" >
             <bool>false</bool>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formats_tab" >
       <attribute name="title" >
        <string>Supported formats</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QTreeWidget" name="treeWidget" >
            <column>
             <property name="text" >
              <string/>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>Value</string>
             </property>
            </column>
            <item>
             <property name="text" >
              <string>New Item</string>
             </property>
             <property name="text" >
              <string/>
             </property>
             <item>
              <property name="text" >
               <string>New Sub Item</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </item>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="controls_tab" >
       <attribute name="title" >
        <string>Controls</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <widget class="QScrollArea" name="scrollArea" >
          <property name="widgetResizable" >
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>440</width>
             <height>233</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3" >
            <item row="0" column="0" >
             <layout class="QGridLayout" name="controlsGridLayout" >
              <property name="horizontalSpacing" >
               <number>10</number>
              </property>
              <property name="verticalSpacing" >
               <number>4</number>
              </property>
              <item row="0" column="0" >
               <widget class="QLabel" name="label_11" >
                <property name="text" >
                 <string>brightness</string>
                </property>
                <property name="wordWrap" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QSlider" name="horizontalSlider" >
                <property name="minimum" >
                 <number>4</number>
                </property>
                <property name="maximum" >
                 <number>70</number>
                </property>
                <property name="singleStep" >
                 <number>2</number>
                </property>
                <property name="value" >
                 <number>50</number>
                </property>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="2" >
               <widget class="QLabel" name="label_10" >
                <property name="text" >
                 <string>10 (0/100)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label_12" >
                <property name="text" >
                 <string>horizontal flip</string>
                </property>
                <property name="wordWrap" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QCheckBox" name="checkBox" >
                <property name="text" >
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="0" >
               <widget class="QLabel" name="label_13" >
                <property name="text" >
                 <string>vertical flip</string>
                </property>
                <property name="wordWrap" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1" >
               <widget class="QCheckBox" name="checkBox_2" >
                <property name="text" >
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAboutApp" />
    <addaction name="actionAboutQT" />
   </widget>
   <widget class="QMenu" name="menuDevice" >
    <property name="title" >
     <string>Device</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionClose" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuSetings" >
    <property name="title" >
     <string>Settings</string>
    </property>
    <addaction name="actionRefresh" />
    <addaction name="actionRefreshAutomaticaly" />
   </widget>
   <addaction name="menuDevice" />
   <addaction name="menuSetings" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionAboutApp" >
   <property name="text" >
    <string>About Application</string>
   </property>
  </action>
  <action name="actionAboutQT" >
   <property name="text" >
    <string>About QT</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open Device</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRefreshAutomaticaly" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Refresh Automaticaly</string>
   </property>
  </action>
  <action name="actionRefresh" >
   <property name="text" >
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close Device</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
